### ISSUE TITLE
BUG: rowLabels and colLabels args in pandas.plotting.table have no effect

### ISSUE THREAD

**Issue by Ivan-basis** [pandas-dev/pandas#37811]
- [ yes ] I have checked that this issue has not already been reported.

- [ yes ] I have confirmed this bug exists on the latest version of pandas.

- [ yes ] (optional) I have confirmed this bug exists on the master branch of pandas.

---

```python
# Your code here
import pandas as pd
from matplotlib import pyplot as plt

df = pd.DataFrame({'col_title':[42,73]})
fig,ax = plt.subplots(1,1)
pd.plotting.table(ax,df, colLabels=['hello'], loc='center')
ax.axis('off')
plt.show()
```

#### Problem description

Setting `rowLabels `or `colLabels `in `pandas.plotting.table` has no effect. I've found that the problem is in `table() `function of **pandas/plotting/_misc.py** as it returns plot_backend.table with predefined rowLabels=None, colLabels=None:
```
    return plot_backend.table(
        ax=ax, data=data, rowLabels=None, colLabels=None, **kwargs
    )
```
It obviously should just pass `rowLabels `and `colLabels ` as is, instead.
Just get rid of None predefinitions in line 26, please.

#### Expected Output

Expected output is a table with column named 'hello' instead of 'col_name'

#### Output of ``pd.show_versions()``

<details>

INSTALLED VERSIONS
------------------
commit           : 67a3d4241ab84419856b84fc3ebc9abcbe66c6b3
python           : 3.7.9.final.0
python-bits      : 64
OS               : Windows
OS-release       : 10
Version          : 10.0.19041
machine          : AMD64
processor        : Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
byteorder        : little
LC_ALL           : None
LANG             : en
LOCALE           : None.None

pandas           : 1.1.4
numpy            : 1.19.4
pytz             : 2020.4
dateutil         : 2.8.1
pip              : 20.2.4
setuptools       : 49.6.0.post20201009
Cython           : 0.29.21
pytest           : None
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : 2.11.2
IPython          : 7.19.0
pandas_datareader: None
bs4              : None
bottleneck       : None
fsspec           : None
fastparquet      : None
gcsfs            : None
matplotlib       : 3.3.2
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : None
pytables         : None
pyxlsb           : None
s3fs             : None
scipy            : 1.5.3
sqlalchemy       : None
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
numba            : None

</details>

**Comment by rhshadrach** [comment]
Thanks for the report @Ivan-basis! Would you be interested in submitting a PR to fix? It also looks like the docs can be improved here, as the descriptions of rowLables and colLabels only appears under the kwargs instead their own entries in the docstring.

**Comment by ma3da** [comment]
Hi @Ivan-basis, are you interested in writing the fix?

**Comment by Ivan-basis** [comment]
@ma3da yes, thank you! I'll make a pr from my base account soon. It's my first pr, so it takes time a little)
